RELEASE_NOTES.md

```diff
@@ -1,5 +1,116 @@
 # Release Notes
 
+## 4.2.0
```

Comment:
@TomSweeneyRedHat PTAL at the release notes if you get a chance

Reply:
Looking now, until dinner, then will wrap after.